worlds/stardew_valley/docs/en_Stardew Valley.md

```diff
@@ -18,35 +18,49 @@ The player can choose from a number of goals, using their YAML settings.
 - Succeed [Grandpa's Evaluation](https://stardewvalleywiki.com/Grandpa) with 4 lit candles
 - Reach the bottom of the [Pelican Town Mineshaft](https://stardewvalleywiki.com/The_Mines)
 - Complete the [Cryptic Note](https://stardewvalleywiki.com/Secret_Notes#Secret_Note_.2310) quest, by meeting Mr Qi on floor 100 of the Skull Cavern
-- Get the achievement [Master Angler](https://stardewvalleywiki.com/Fish), which requires catching every fish in the game
-- Get the achievement [A Complete Collection](https://stardewvalleywiki.com/Museum), which requires donating all the artifacts and minerals to the museum
-- Get the achievement [Full House](https://stardewvalleywiki.com/Children), which requires getting married and having two kids.
+- Become a [Master Angler](https://stardewvalleywiki.com/Fish), which requires catching every fish in your slot
+- Restore [A Complete Collection](https://stardewvalleywiki.com/Museum), which requires donating all the artifacts and minerals to the museum
+- Get the achievement [Full House](https://stardewvalleywiki.com/Children), which requires getting married and having two kids
 - Get recognized as the [Greatest Walnut Hunter](https://stardewvalleywiki.com/Golden_Walnut) by Mr Qi, which requires finding all 130 golden walnuts on ginger island
+- Become the [Protector of the Valley](https://stardewvalleywiki.com/Adventurer%27s_Guild#Monster_Eradication_Goals) by completing all the monster slayer goals at the Adventure Guild
+- Complete a [Full Shipment](https://stardewvalleywiki.com/Shipping#Collection) by shipping every item in your slot
+- Become a [Gourmet Chef](https://stardewvalleywiki.com/Cooking) by cooking every recipe in your slot
+- Become a [Craft Master](https://stardewvalleywiki.com/Crafting) by crafting every item
+- Earn the title of [Legend](https://stardewvalleywiki.com/Gold) by earn 10 000 000g 
+- Solve the [Mystery of the Stardrops](https://stardewvalleywiki.com/Stardrop) by find every stardrop 
```

Comment:
```suggestion
- Solve the [Mystery of the Stardrops](https://stardewvalleywiki.com/Stardrop) by finding every stardrop 
```

Reply:
He’s Quebecois so it makes some sense.

Reply:
It's typos, not me being unable to speak english lmao